cmake/modules/ddr/cmake_ddr.awk

```diff
@@ -78,9 +78,11 @@ function write_flag_defined(macro_name) {
 }
 
 function define_flag_macro(macro_name) {
-	# TODO if not include guard
-	write_flag_macro(macro_name);
-	write_flag_defined(macro_name);
+	# Don't process any include guard macros
```

Comment:
I assume the old makefiles were doing something similar?

Reply:
yes
https://github.com/eclipse/omr/blob/2d1e78503606e86fc9751846a252380da8d2e9fe/ddr/tools/getmacros#L116